app/(candidate)/dashboard/components/tasks/flows/TaskFlow.js

```diff
@@ -282,7 +283,7 @@ export default function TaskFlow({
         {stepName === STEPS.purchase && (
           <PurchaseIntentProvider
             {...{
-              type: type.toUpperCase(),
+              type: (type === 'p2p' ? 'text' : type).toUpperCase(),
```

Comment:
Just change this to `p2p` and use the constant as well.

Reply:
@RavenHursT I think we want to keep it as text (using purchase type constant) because the purchase type will stay the same, just the outreach type changing to p2p...right?